cmd/opm/registry/serve.go

```diff
@@ -98,3 +111,17 @@ func runRegistryServeCmdFunc(cmd *cobra.Command, args []string) error {
 
 	return nil
 }
+
+func migrateToLatest(db *sql.DB) error {
```

Comment:
I don't think this is the right place for this function. If we want to reuse it in other places (like when we add other commands to `opm registry`, maybe we should move it into the `sqlite` package? Otherwise we should just leave it in line.

Reply:
@kevinrizza this is a stop gap measure till we have the `migrate` command available with the opm cli. Once we have that this function can be removed and the corresponding function from the `registry` package should be used [here](https://github.com/operator-framework/operator-registry/blob/9d0489171c30c887bae1341760584bb967ceef3a/cmd/opm/registry/serve.go#L75)